dvc/output/base.py

```diff
@@ -393,8 +393,10 @@ def get_used_cache(self, **kwargs):
         include the `info` of its files.
         """
 
-        if self.stage.is_repo_import:
-            return []
+        # XXX: There's no test for this one, it might not be needed
```

Comment:
This one is no longer needed because you are collecting repo imports separately in used_cache, so this is not going to get pulled automatically as it was before. So we can safely remove this from here.

Reply:
great, thanks for confirming it, @efiop !